ShortenLink/Scenes/LinksPopOver/LinksPopOverViewController.swift

```diff
@@ -67,7 +116,7 @@ extension LinksPopOverViewController {
         }
 
         scrollView.snp.makeConstraints {
-            $0.top.equalTo(titleField.snp.bottom).inset(-10.0)
+            $0.top.equalTo(titleField.snp.bottom).offset(8.0)
             $0.trailing.leading.bottom.equalToSuperview()
         }
```

Comment:
I think we should add scrollView after user is signed in only so we should add it together with ShortenLinkViewController. What do you think.